cypress/integration/locators.spec.ts

```diff
@@ -0,0 +1,35 @@
+import { PersonalInformationPage } from '../page';
+
+describe('Given a page to practice automation', () => {
+  before(() => {
+    cy.visit('http://toolsqa.com/automation-practice-form/');
+  });
+
+  describe('when I am training locators', () => {
+    const personalInformationPage = new PersonalInformationPage();
+
+    before(() => {
+      debugger;
```

Comment:
remove debugger key word, this is only used for local debug in js.

Reply:
Done